core/src/main/java/org/opensearch/sql/data/model/ExprDatetimeValue.java

```diff
@@ -16,6 +16,7 @@
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeFormatterBuilder;
 import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
```

Comment:
what's this?

Reply:
It's used to validate dates.

Reply:
In this file?  It's being imported but I don't see where you've used the import.